llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -1362,17 +1360,7 @@ def compute_loss_from_logits(
     else:
         loss = losses.sum() / (targets != loss_fn.ignore_index).sum()
         if sample_weighing_factor is not None:
```

Comment:
since arg was deprecated and is now being removed, you should just remove the arg.

Reply:
Sounds good, done